### ISSUE TITLE
Add option to remove a node from the database after it has been shutdown forever

### ISSUE THREAD

**Issue by marcanpilami** [enioka/jqm#216]
To be discussed: In admin ? In CLI?

**Comment by digitalfox** [comment]
Where it is easiest to code :-). That's a feature not used that much. And frankly, even a simple SQL snippets in documentation could be enough.

**Comment by marcanpilami** [comment]
I agree this is an edge requirement. But it would be interesting to add to the new CLI described in #118 

That being said, I believe we should never allow the metadata to be directly updated inside the database by the users - this data should always be handled through the dedicated APIs which allow for control and always check for consistency. So no workaround for now ;-)

**Comment by Pauleing** [comment]
In frontend add a button to delete a node that has been shut down for [X] hours (use `lastSeenAlive` attribute)
